pandas/core/indexes/datetimelike.py

```diff
@@ -572,63 +669,6 @@ def wrap_arithmetic_op(self, other, result):
     return result
 
 
-def wrap_array_method(method, pin_name=False):
-    """
-    Wrap a DatetimeArray/TimedeltaArray/PeriodArray method so that the
-    returned object is an Index subclass instead of ndarray or ExtensionArray
-    subclass.
-
-    Parameters
-    ----------
-    method : method of Datetime/Timedelta/Period Array class
-    pin_name : bool
```

Comment:
it looks like pin_name hasn't been replaced anywhere.  Has this become an always-do or a never-do?